timeless/sync/celery.py

```diff
@@ -0,0 +1,18 @@
+from celery import Celery
+
+
+def make_celery(app):
```

Comment:
@paulodamaso Let's also import this in `timeless/__init__.py` and create a celery instance. To make it ready to use.